src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -131,6 +131,7 @@ public static class AccessibilityHelpers
             new Browser("com.jerky.browser2", "enterUrl"),
             new Browser("com.ksmobile.cb", "address_bar_edit_text"),
             new Browser("com.linkbubble.playstore", "url_text"),
+            new Browser("com.lemurbrowser.exts","url_bar"),
```

Comment:
Please move this up one line to ensure the sections are sorted alphabetically as mentioned on [line 26](https://github.com/bitwarden/mobile/pull/2262/files#diff-4f8cec091436d3dd5e83798a9727a121a81a82a90766200d840966a6b3ea9bc5L26)

Reply:
already move this up one line. thank you so much to mention it.